db/migrate/20180112125015_fix_taxable_taxonomies_job_template.rb

```diff
@@ -0,0 +1,9 @@
+class FixTaxableTaxonomiesJobTemplate < ActiveRecord::Migration
```

Comment:
needs a version added now.

Reply:
Updated, tested originally with Foreman 1.15, therefore missed this.